web/tests/libtest/env.py

```diff
@@ -257,15 +257,20 @@ def parts_to_url(codechecker_cfg):
 
 
 def get_workspace(test_id='test'):
-    tmp_dir = os.path.join(REPO_ROOT, 'build')
-    base_dir = os.path.join(tmp_dir, 'workspace')
-    if not os.path.exists(base_dir):
-        os.makedirs(base_dir)
+    """ return a temporary workspace for the tests """
+    workspace_root = os.environ.get("WORKSPACE")
+    if not workspace_root:
+        # if no external workspace is set create under the build dir
+        tmp_dir = os.path.join(REPO_ROOT, 'build')
```

Comment:
This `tmp_dir` variable can be eliminated because we use it only in one place. You can use this:
```py
workspace_root = os.path.join(REPO_ROOT, 'build', 'workspace')
```